platform/nrf52/drivers/componolit-runtime-drivers-gpio.adb

```diff
@@ -11,22 +16,28 @@ is
    Outset : Pin_Values with
       Address => SSE.To_Address (AHB_Base + OUTSET_Offset),
       Import,
-      Volatile;
+      Volatile,
+      Async_Readers,
+      Effective_Writes;
 
    Outclr : Pin_Values with
       Address => SSE.To_Address (AHB_Base + OUTCLR_Offset),
       Import,
-      Volatile;
+      Volatile,
+      Async_Readers,
+      Effective_Writes;
 
    Outreg : Pin_Values with
       Address => SSE.To_Address (AHB_Base + OUT_Offset),
       Import,
-      Volatile;
+      Volatile,
+      Async_Writers;
 
    Inreg : Pin_Values with
       Address => SSE.To_Address (AHB_Base + IN_Offset),
       Import,
-      Volatile;
+      Volatile,
+      Async_Writers;
```

Comment:
The register naming is somewhat inconsistent. While `Outset` is identical to the spec (minus casing), `Outreg` isn't (because it would conflict with the `out` keyword, just like `Inreg`). Maybe we should suffix all of them with `_Reg`? Also, maybe we want to leave the casing as in the spec (e.g. `OUTCLR_Reg`)?

Reply:
Fixed in fe176b21c1d1ffb0a3531a48c3556a92c11d6935